### PR TITLE
Fix soundmeter for SFU

### PR DESCRIPTION
**Pull request by lmangani**
Reattaching the soundMeter script to the SFU stream object.
TODO: emit event to mesh @Dletta

### CODE REVIEW

File: src/assets/js/sfu/sfu.js [hunk 1]
```diff
@@ -54,6 +54,19 @@ export default class SFU extends EventEmitter {
...
         await this.sfuRoom.sendAudio(video.srcObject.getAudioTracks()[0]);
+        
+        // Attach SoundMeter to Local Stream
+        if (SoundMeter) {
+          // Soundmeter
+          window.stream = video.srcObject;
+          // console.log('Init Soundmeter.........');
+          const soundMeter = new SoundMeter(function () {
+            console.log('Imm Speaking! Sending metadata mesh focus...');
+            // metaData.sendControlData({ username: username, id: socketId, talking: true });
```
Comment (lmangani) [inline src/assets/js/sfu/sfu.js:65]: @Dletta could you patch with the right emitter class?

File: src/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
   <script src="/assets/js/observer.js" type="text/javascript"></script>
   <script src="/assets/js/tingle.js" type="text/javascript"></script>
-  <script type="module" src="/assets/js/soundmeter.js"></script>
+  <script src="/assets/js/soundmeter.js" type="text/javascript"></script>
```
Comment (QVDev) [inline src/index.html:27]: If soundmeter is a module than we dont need this we can import it directly

File: src/assets/js/sfu/sfu.js [hunk 3]
```diff
@@ -54,6 +54,19 @@ export default class SFU extends EventEmitter {
         const video = this.config.localVideoEl;
         await this.sfuRoom.sendVideo(video.srcObject.getVideoTracks()[0]);
         await this.sfuRoom.sendAudio(video.srcObject.getAudioTracks()[0]);
+        
+        // Attach SoundMeter to Local Stream
+        if (SoundMeter) {
+          // Soundmeter
+          window.stream = video.srcObject;
```
Comment (QVDev) [inline src/assets/js/sfu/sfu.js:61]: Against using window.stream.

But if we want than we can move all magic inside soundmeter and no need to add this to the sfu.js

Comment (lmangani) [reply src/assets/js/sfu/sfu.js]: done

**Pull request merged**